doc/man/scons.xml

```diff
@@ -4955,37 +4907,28 @@ b = Builder(generator=g)
 and
 <emphasis>action</emphasis>
 arguments must not both be used for the same Builder.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>src_builder</term>
   <listitem>
 <para>Specifies a builder to use when a source file name suffix does not match
 any of the suffixes of the builder. Using this argument produces a
 multi-stage builder.</para>
-
   </listitem>
   </varlistentry>
+
   <varlistentry>
   <term>single_source</term>
   <listitem>
 <para>Specifies that this builder expects exactly one source file per call. Giving
 more than one source file without target files results in implicitly calling
 the builder multiple times (once for each source given). Giving multiple
 source files together with target files results in a UserError exception.</para>
-
   </listitem>
   </varlistentry>
-</variablelist>
 
-<para>The
```

Comment:
Did you move this bit somewhere else?

Reply:
Yes. Old way, there was one mention within the section that talks about the generator function, and this one, which was dangling. Dropped this and added the wording in the section on the action function - should be around line 4556.